README.md

```diff
@@ -0,0 +1,44 @@
+## Overview
+
+Provide the type of interaction between the cita-node 
+interaction type and the cita component, use protobuf 
+to generate the corresponding file. And have generated
+files for the following languages in advance:
+- Rust
+- go
+- javascript
+- java
+
+You can use them derectly, and be free to use the `protos` 
```

Comment:
`s/protos/protoc`

Reply:
I mean "use the photo files in protos directory.